content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md

```diff
@@ -1,22 +1,284 @@
+            host: istio-egressgateway.istio-system.svc.cluster.local
+            subset: cnn
+            port:
+              number: 443
+          weight: 100
+      - match:
+        - gateways:
+          - istio-egressgateway
+          port: 443
+        route:
+        - destination:
+            host: edition.cnn.com
+            port:
+              number: 443
+          weight: 100
+    ---
+    apiVersion: networking.istio.io/v1alpha3
+    kind: DestinationRule
+    metadata:
+      name: originate-tls-for-edition-cnn-com
+    spec:
+      host: edition.cnn.com
+      trafficPolicy:
+        loadBalancer:
+          simple: ROUND_ROBIN
+        portLevelSettings:
+        - port:
+            number: 443
+          tls:
+            mode: SIMPLE # initiates HTTPS for connections to edition.cnn.com
+    EOF
+    {{< /text >}}
+
+1.  Send an HTTP request to [http://edition.cnn.com/politics](https://edition.cnn.com/politics).
+
+    {{< text bash >}}
+    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -sL -o /dev/null -D - http://edition.cnn.com/politics
+    HTTP/1.1 200 OK
+    ...
+    content-length: 150793
+    ...
+    {{< /text >}}
+
+    The output should be the same as in the [TLS Origination for Egress Traffic](/docs/examples/advanced-gateways/egress-tls-origination/)
+    example, with TLS origination: without the _301 Moved Permanently_ message.
+
+1.  Check the log of the `istio-egressgateway` pod and you should see a line corresponding to our request.
+    If Istio is deployed in the `istio-system` namespace, the command to print the log is:
+
+    {{< text bash >}}
+    $ kubectl logs $(kubectl get pod -l istio=egressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') egressgateway -n istio-system | tail
+    {{< /text >}}
+
+    You should see a line similar to the following:
+
+    {{< text plain>}}
+    "[2018-06-14T13:49:36.340Z] "GET /politics HTTP/1.1" 200 - 0 148528 5096 90 "172.30.146.87" "curl/7.35.0" "c6bfdfc3-07ec-9c30-8957-6904230fd037" "edition.cnn.com" "151.101.65.67:443"
+    {{< /text >}}
+
+### Cleanup the TLS origination example
+
+Remove the Istio configuration items you created:
+
+{{< text bash >}}
+$ kubectl delete gateway istio-egressgateway
+$ kubectl delete serviceentry cnn
+$ kubectl delete virtualservice direct-cnn-through-egress-gateway
+$ kubectl delete destinationrule originate-tls-for-edition-cnn-com
+$ kubectl delete destinationrule egressgateway-for-cnn
+{{< /text >}}
+
+## Perform mutual TLS origination with an egress gateway
+
+Similar to the previous section, this section describes how to configure an egress gateway to perform
+TLS origination for an external service, only this time using a service that requires mutual TLS.
+
+This example is considerably more involved because you need to first:
+
+1. generate client and server certificates
+1. deploy an external service that supports the mTLS protocol
+1. redeploy the egress gateway server with the needed mTLS certs
```

Comment:
why "egress gateway server" and not just "egress gateway"? or maybe "egress gateway pod" or "egress gateway component".